libs/dh/shared/data-access-mocks/src/lib/process/data.ts

```diff
@@ -0,0 +1,126 @@
+//#region License
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//#endregion
+/* eslint-disable sonarjs/no-duplicate-string */
+import {
+  CalculationExecutionType,
+  CalculationType,
+  MeteringPointType,
+  PriceType,
+  ProcessState,
+  ProcessStepState,
+} from '@energinet-datahub/dh/shared/domain/graphql';
+
+export default [
+  {
+    __typename: 'ElectricalHeatingCalculation' as const,
+    id: '1',
+    startedAt: new Date('2022-01-01T00:00:00Z'),
+    calculationType: CalculationType.Aggregation,
+    createdAt: new Date('2022-01-01T00:00:00Z'),
+    state: ProcessState.Running,
+    steps: [
+      {
+        __typename: 'OrchestrationInstanceStep' as const,
+        isCurrent: true,
+        state: ProcessStepState.Running,
+      },
+    ],
```

Comment:
suggestion: the `as const` casting looks a bit off. If we want to keep the `export default` pattern, then it's better to assign the array to a typed constant and export it afterwards. Or was there another reason for using `as const`?

```ts
const data: ElectricalHeatingCalculation[] = [ ... ];

export default data;
```

Reply:
It is to make typescript happy. If __typename is not a const. Typescript fails to infer the right type after the data is imported.

Reply:
Fixed